src/Application.php

```diff
@@ -192,6 +196,18 @@ public function configure(): ContainerInterface
             }
         }
 
+        $tearDown = function () use ($container): bool {
```

Comment:
no longer needs to be in a var, I was also looking at exception handler but method sig is different and also it wouldn't have worked anyway due to current exception handling IIRC